### PR TITLE
feat(tasks): enable approval task actions

### PR DESCRIPTION
**Pull request by alexkrolick**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @alexkrolick has signed the CLA. Thanks for the pull request!

File: src/elements/content-sidebar/activity-feed/task-new/Task.js [hunk 1]
```diff
@@ -74,6 +75,13 @@ class Task extends React.Component<Props> {
         const hiddenAssigneeCount = assigneeCount - MAX_AVATARS;
         const isOverdue = due_at ? status === TASK_NEW_INCOMPLETE && new Date(due_at) < Date.now() : false;
 
+        const shouldShowActions =
+            currentUserAssignment &&
+            currentUserAssignment.permissions &&
+            currentUserAssignment.permissions.can_update &&
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/task-new/Task.js]: Looks like this could benefit from `lodash/get`.

Comment (alexkrolick) [reply src/elements/content-sidebar/activity-feed/task-new/Task.js]: Yeah, except get mangles Flow, so you can't validate the object you are parsing is valid.

Comment (jstoffan) [reply src/elements/content-sidebar/activity-feed/task-new/Task.js]: Would the following work?

```
const canUpdate: boolean = getProp(currentUserAssignment, 'permissions.can_update', false);
```

Comment (alexkrolick) [reply src/elements/content-sidebar/activity-feed/task-new/Task.js]: Yes, but Flow won't complain if you put in e.g., `permissions.can_kjhasdf` into `_.get`, but I want it to.

**Pull request merged**